cssom/Overview.bs

```diff
@@ -1457,10 +1462,41 @@ To <dfn>serialize a CSS rule</dfn>, perform one of the following in accordance w
 
 
  <dt>{{CSSMediaRule}}
- <dd>Issue: ...
+ <dd>
+  The result of concatenating the following:
+  <ol>
+   <li>The string "<code>@media</code>", followed by a single SPACE (U+0020).</li>
+   <li>The result of performing <a>serialize a media query list<a> on rule's media query list.</li>
+   <li>A single SPACE (U+0020), followed by the string "{", i.e., LEFT CURLY BRACKET (U+007B), followed by a newline.</li>
+   <li>The result of performing <a>serialize a CSS rule</a> on the each rule, seperated by a newline and indented by two spaces, in rule's CSSRules list.</li>
+   <li>A newline, followed by the string "}", i.e., RIGHT CURLY BRACKET (U+007D)</li>
+  </ol>
 
  <dt>{{CSSFontFaceRule}}
- <dd>Issue: ...
+ <dd>
+  The result of concatenating the following:
+  <ol>
+   <li>The string "<code>@font-face {</code>", followed by a single SPACE (U+0020).</li>
+   <li>The string "<code>font-family:</code>", followed by a single SPACE (U+0020).</li>
+   <li>The result of performing <a>serialize a string</a> on the rule’s font family name.</li>
+   <li>The string "<code>;</code>", i.e., SEMICOLON (U+003B).
+   <li>
+    If the rule's associated source list is not empty, follow these substeps:
+    <ol>
+     <li>A single SPACE (U+0020), followed by the string "<code>src:</code>", followed by a single SPACE (U+0020).</li>
+     <li>The result of invoking <a>serialize a comma-separated list</a> on performing <a>serialize a URL</a> or <a>serialize a LOCAL<a> for each source on the source list.</li>
+     <li>The string "<code>;</code>", i.e., SEMICOLON (U+003B).</li>
+    </ol>
+   </li>
+   <li>If rule's associated <code>unicode-range</code> descriptor is present, a single SPACE (U+0020), followed by the string "<code>unicode-range:</code>", followed by a single SPACE (U+0020), followed by the result of performing serialize a <unicode-rule>, followed by the string "<code>;</code>", i.e., SEMICOLON (U+003B).</li>
```

Comment:
I wrote them as `the result of performing serialize a <unicode-rule>` because they aren't in exist in current spec. Should I make something different?